packages/eslint-plugin-ecmascript-compat/lib/features/es2020.spec.js

```diff
@@ -22,7 +22,26 @@ const ruleTester = new RuleTester({
 });
 
 ruleTester.run('compat', require('../rule'), {
-  valid: [],
+  valid: [
+    {
+      code: 'globalThis.foo;',
+      options: [{ polyfills: ['globalThis'] }],
+    },
+    {
+      code: 'Promise.allSettled();',
+      options: [{ polyfills: ['Promise.prototype.allSettled'] }],
+      errors: [{ message: "ES2020 'Promise.allSettled' function is forbidden." }],
+    },
```

Comment:
`errors` left over after copy paste above and below here?

Reply:
unfortunately yes. i'll fix it.